docs/integrations/sources/google-search-console.md

```diff
@@ -98,6 +101,10 @@ The Google Search Console Source connector supports the following [ sync modes](
    The granularity for the cursor is 1 day, so Incremental Sync in Append mode may result in duplicating the data.
 :::
 
+:::note
+    Parameter `data_state='all'` should not be used with Incremental Sync mode as it may cause data loss.
```

Comment:
I believe it would be beneficial to include this information in the field description within the spec file.